Userland/Libraries/LibCore/System.cpp

```diff
@@ -442,7 +442,7 @@ ErrorOr<String> ptsname(int fd)
 {
     auto* name = ::ptsname(fd);
     if (!name)
-        return Error::from_syscall("ptsname"sv, -errno);
+        return Error::from_string_literal("Could not find pts"sv);
```

Comment:
We already have many other things using `Error::from_syscall()` that aren't *actually* kernel system calls. So I think it's better to leave this alone instead of making the error slightly worse.